README.md

```diff
@@ -2,28 +2,6 @@
 
 ## Getting started
 
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
-
-### On macOS and Linux
-
-```bash
-$ source setup.sh
-```
-
-### On Windows
-
-Git Bash:
-
-```bash
-$ source setup.sh --windows
-```
-
-PowerShell
-
-```bash
-$ ./setup --windows
-```
-
 ### Environment variables
 
 To run effectively, update the `.env` file with your application details:
```

Comment:
Note that the `.env` file needs to be copied from `.env.template` for this to work

Also `.env.template` still has the old value of `FLASK_APP` (`app` and not `todo_app/app`)

Reply:
Updated in 26bb5cb1c97a932218a772fde560fac452a02bfc